pkg/odo/cli/component/create.go

```diff
@@ -172,6 +173,7 @@ A full list of component types that can be deployed is available using: 'odo cat
 			os.Exit(1)
 		}
 
+		fmt.Printf("Initializing '%s' component ...\n", componentName)
```

Comment:
You should have it like this instead:
`log.Successf("Initializing '%s' component", componentName)`

No newline at the end or the `...`